docs/using-guidelines.asciidoc

```diff
@@ -1,11 +1,25 @@
-//[[ecs-guidelines]]
-== Guidelines and Best Practices
+[[ecs-guidelines]]
+=== Guidelines and Best Practices
 
 The {ecs} schema serves best when you follow schema guidelines and best
 practices.
 
 [float]
-=== General guidelines
+==== Types of fields
+
+ECS defines "Core" and "Extended" fields.
```

Comment:
Again, should mention that these are "levels"  indicated in the levels column, and these should be consistent with those listed in the converting asciidoc file.